config/values/recruitmentcycle.yml

```
@@ -0,0 +1,8 @@
+recruitmentcycle:
+  coursestart: September 2025
+  alltimesanddates: from 9am on 10 October 2024 to 6pm on 16 September 2025
```

```suggestion
  openanddeadline: from 9am on 8 October 2024 to 6pm on 16 September 2025
```
worth being a bit clearer about what this is?

also - more importantly! - isn't this supposed to be **8** October 2024?